plugins/services/src/js/components/forms/ContainerServiceFormSection.js

```diff
@@ -169,23 +139,28 @@ class ContainerServiceFormSection extends Component {
       );
 
       if (containerType === DOCKER) {
-        dockerOnly = '';
-      } else {
-        label = label + ' ';
+        dockerOnly = null;
       }
 
       return (
-        <FieldLabel key={index}>
-          <FieldInput
-            checked={containerType === DOCKER && Boolean(checked)}
-            name={`${path}.docker.${settingName}`}
-            type="checkbox"
-            disabled={containerType !== DOCKER}
-            value={settingName} />
-          {label}
-          {this.getTooltipIfContent(dockerOnly)}
-          <FieldHelp>{helpText}</FieldHelp>
-        </FieldLabel>
+        <FormRow>
+          <FormGroup className="column-12">
+            <FieldLabel key={index} wordWrap>
```

Comment:
As it turns out, when you're setting boolean props to `true` in JSX, you don't need to provide the value. Similar to HTML's `disabled`.